src/components/Exercises/ExercisesListItem/ExercisesListItem.js

```diff
@@ -64,8 +62,29 @@ const ExercisesListItem = ({
       <DifficultyIcon difficulty={difficulty} />
     </td>
     <td className="text-nowrap">
-      <FormattedDate value={createdAt * 1000} />{' '}
-      <FormattedTime value={createdAt * 1000} />
+      <OverlayTrigger
+        placement="right"
+        overlay={
+          <Tooltip id={`created-tooltip-${id}`}>
+            <span>
+              <FormattedMessage
+                id="generic.created"
+                defaultMessage="Created"
+              />:
+              <FormattedDate value={createdAt * 1000} />{' '}
+              <FormattedTime value={createdAt * 1000} />
+              <br />
+              <FormattedMessage id="generic.updated" defaultMessage="Updated" />
+              :<FormattedDate value={createdAt * 1000} />{' '}
```

Comment:
The message says `updated` but the variable is `created` ... is this intentional?